cypress/integration/pages/Forgot/index.test.js

```diff
@@ -0,0 +1,28 @@
+describe("Page Forgot", function() {
+  it("Verify if exist the field E-mail", function() {
+    cy.visit("/auth/forgot");
+
+    cy.contains("E-mail:");
+    cy.get("input[name=email]");
+  });
+
+  it("Send form without filling in the inputs", function() {
+    cy.visit("/auth/forgot");
+    cy.contains("Enviar").click();
+
+    cy.contains("E-mail é obrigatório");
+  });
+
+  it("Send form with email invalid", function() {
+    cy.visit("/auth/forgot");
+    cy.get("input[name=email]").type("empix");
+    cy.contains("Enviar").click();
+    cy.contains("Preencha com email válido");
+  });
+
+  it("Send form with all fields valid", function() {
+    cy.visit("/auth/forgot");
+    cy.get("input[name=email]").type("any.email@gmail.com");
+    cy.contains("Enviar").click();
+  });
```

Comment:
This test doesn't is verifying if the form will is filled with all the fields valid. It's missing something that has this verification, something that verifies if the user is on the next page or if it appeared a success message.

Reply:
I forgot. I will change that.
Thanks.